service/ui.go

```diff
@@ -311,21 +311,21 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 	app.AddRoute("/task/{task_id}").Wrap(needsLoginToggleable, needsContext, viewTasks).Handler(uis.taskPage).Get()
 	app.AddRoute("/task/{task_id}/{execution}").Wrap(needsLoginToggleable, needsContext, viewTasks).Handler(uis.taskPage).Get()
 	app.AddRoute("/tasks/{task_id}").Wrap(needsLogin, needsContext, editTasks).Handler(uis.taskModify).Put()
-	app.AddRoute("/json/task_log/{task_id}").Wrap(needsLoginToggleable, needsContext, viewLogs).Handler(uis.taskLog).Get()
-	app.AddRoute("/json/task_log/{task_id}/{execution}").Wrap(needsLoginToggleable, needsContext, viewLogs).Handler(uis.taskLog).Get()
-	app.AddRoute("/task_log_raw/{task_id}/{execution}").Wrap(needsLoginToggleable, needsContext, allowsCORS, viewLogs).Handler(uis.taskLogRaw).Get()
+	app.AddRoute("/json/task_log/{task_id}").Wrap(needsLogin, needsContext, viewLogs).Handler(uis.taskLog).Get()
+	app.AddRoute("/json/task_log/{task_id}/{execution}").Wrap(needsLogin, needsContext, viewLogs).Handler(uis.taskLog).Get()
+	app.AddRoute("/task_log_raw/{task_id}/{execution}").Wrap(needsLogin, needsContext, allowsCORS, viewLogs).Handler(uis.taskLogRaw).Get()
 
 	// Performance Discovery pages
 	app.AddRoute("/perfdiscovery/").Wrap(needsLogin, needsContext).Handler(uis.perfdiscoveryPage).Get()
 	app.AddRoute("/perfdiscovery/{project_id}").Wrap(needsLogin, needsContext, viewTasks).Handler(uis.perfdiscoveryPage).Get()
 
 	// Test Logs
-	app.AddRoute("/test_log/{log_id}").Wrap(needsLoginToggleable, needsContext, allowsCORS).Handler(uis.testLog).Get()
-	app.AddRoute("/test_log/{task_id}/{task_execution}").Wrap(needsLoginToggleable, needsContext, allowsCORS, viewLogs).Handler(uis.testLog).Get()
+	app.AddRoute("/test_log/{log_id}").Wrap(needsLogin, needsContext, allowsCORS).Handler(uis.testLog).Get()
+	app.AddRoute("/test_log/{task_id}/{task_execution}").Wrap(needsLogin, needsContext, allowsCORS, viewLogs).Handler(uis.testLog).Get()
```

Comment:
As a sanity check can you please ensure parsley can still reach this route in staging? Opening a test log in staging should be sufficient.